### PR TITLE
Add module mail_full_expand

### PR DESCRIPTION
**Pull request by yajo**
Move mail_full_expand from [odoo-grupoesoc-addons](https://github.com/grupoesoc/odoo-grupoesoc-addons).

This module allows the user to see the full contents of an email. See README for details.

### CODE REVIEW

**Comment by Yajo** [comment]
[Travis seems wrong](https://travis-ci.org/OCA/social/jobs/80198830#L162). No need to add that header in empty files.

**Comment by Yajo** [comment]
Ah no, it seems whitespace garbage.

File: mail_full_expand/README.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Odoo automatically tries to remove blockquotes and signatures from received
+mails. That is useful because it removes lots of distraction, but sometimes it
+removes important information.
+
+Also, messages are narrow to fit in the conversations views, but sometimes you
+receive a mail with predefined width and cannot read it.
+
+This module adds a button to all messages to read them in a floating window
+with its full contents.
```
Comment (pedrobaeza) [inline mail_full_expand/README.rst]: s/its/their

File: mail_full_expand/README.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+* https://www.odoo.com/forum/help-1
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/social/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/social/issues/new?body=module:%20mail_full_expand%0Aversion:%208.0.3.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline mail_full_expand/README.rst]: Put Odoo version (8.0), not module version

**Comment by pedrobaeza** [comment]
Please remove or replace the company icon.

**Comment by dreispt** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**